src/components/notification/notification.jsx

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+import { Avatar } from "../avatar/avatar";
+
+/**
+ * Notification Component
+ * @author [J. Hartsek](https://github.com/JHartsek)
+ * @param src the avatar image link
+ * @param alt the avatar image description
+ * @param title the notificaiton origin
+ * @param summary the notfication details
+ * @param status expects 'read' or 'unread'
+ */
+
+const NotificationWrapper = ({ children }) => ({ children });
+
+NotificationWrapper.Container = styled.div`
+  column-gap: 8px;
+  padding: 12px 0 16px 12px;
+`;
+
+NotificationWrapper.Info = styled.div`
+  row-gap: 4px;
+`;
+
+NotificationWrapper.Title = styled.h3`
+  margin: 0;
+  color: rgba(246, 246, 246, 0.7);
```

Comment:
@jahorwitz When I use the bootstrap classname, it's showing up as fully opaque white instead of the greyish white. It's doing this in storybook and the browser. Do you run into the same issue? 
![Screen Shot 2023-01-27 at 11 56 48 PM](https://user-images.githubusercontent.com/95091979/215243015-8db6b801-bc1b-4324-b48f-70aa979770be.png)
![Screen Shot 2023-01-27 at 11 57 21 PM](https://user-images.githubusercontent.com/95091979/215243017-5b1d10ee-853f-4b27-8e3a-cc6efe3bd5ab.png)

Reply:
Oh weird, I only tried it with `color: var(--bs-white75)` in the styled component but yeah that happens for me to. Looks like this might be one of the differences between bootstrap 4 and 5 that I didn't realize was there. Here's the explanation: https://getbootstrap.com/docs/5.1/utilities/colors/#example

Probably in the future I should update our theme to add opacities for each of our white colors instead of adding theme colors like I did, so for now just use the styled component with `color: var(--bs-white75)`. Good catch!

It may be slightly lighter than the mock, you can also try it with white60 and see if you like that better; I'll leave that up to you